### PR TITLE
Show label and icon for all Widgets

### LINKED ISSUE #3421: In Chart element `icon=` is ignored
The documentation at https://www.openhab.org/docs/ui/sitemaps.html#element-type-chart for the Chart element in sitemaps contains the optional parameters `[icon="<iconname>"] [label="<labelname>"]`.

When the parameters are provided, they are not displayed in openhab-Android 3.6.2-beta/OH 3.4.2.

The same applies to Basic UI: https://github.com/openhab/openhab-webui/issues/1931

### PR DESCRIPTION
**Pull request by mueller-ma**
Closes #3421

### CODE REVIEW

**Comment by lolodomo** [comment]
By curiosity, can you show how it looks for image, video, mapview and chart ?

**Comment by mueller-ma** [comment]
Mapview already does display the title in a row above the actual map. I'd design other widgets (video, image, etc.) similar.

File: mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt [hunk 1]
```diff
@@ -453,16 +454,17 @@ class WidgetAdapter(
...
+    ) : LabeledItemBaseViewHolder(initData, layoutResId, compactModeLayoutResId) {
         protected val widgetContentView: View = itemView.findViewById(R.id.widget_content)
         private val dataSaverView: View = itemView.findViewById(R.id.data_saver)
         private val dataSaverButton: Button = itemView.findViewById(R.id.data_saver_button)
         private val dataSaverHint: TextView = itemView.findViewById(R.id.data_saver_hint)
 
         override fun bind(widget: Widget) {
-            boundWidget = widget
+            super.bind(widget)
+            val showLabelAndIcon = widget.label.isNotEmpty()
```
Comment (mueller-ma) [inline mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt:465]: Show the label and icon if the label is empty or if label and icon are empty?

Comment (mueller-ma) [reply mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt:465]: Keep it consistent with Basic UI: Label must be empty to hide the label and icon.

**Pull request merged**